tests/integration/utils/chroma_log_collector.py

```diff
@@ -177,33 +177,13 @@ def fetch_iml_diagnostics(self, server):
 
         errors = []
 
-        # Copy and expand the diagnostics locally, so they will be
-        # able to be read in browser in Jenkins.
         errors.append(self.fetch_log(server, "/var/tmp/%s" % diagnostics, ""))
 
-        if diagnostics.endswith("tar.xz"):
-            if shell_run(
-                ["tar", "-xvJf", "%s/%s" % (self.destination_path, diagnostics), "-C", self.destination_path]
-            ).rc:
-                errors.append("Error tar --xvJf the iml diagnostics file")
-        elif diagnostics.endswith("tar.gz"):
-            if shell_run(
-                ["tar", "-xvzf", "%s/%s" % (self.destination_path, diagnostics), "-C", self.destination_path]
-            ).rc:
-                errors.append("Error tar -xvzf the iml diagnostics file")
-        else:
-            errors = "Didn't recognize iml-diagnostics file format"
-
-        diagnostics_dir = re.compile("(sosreport-.*)\.tar\..*").search(diagnostics).group(1)
-
-        if shell_run(["chmod", "-R", "777", "%s/%s" % (self.destination_path, diagnostics_dir)]).rc:
+        if shell_run(["chmod", "777", "%s/%s" % (self.destination_path, diagnostics)]).rc:
```

Comment:
With this being diagnostic info this should be fine, just wondering if something like 655 would work?

Reply:
it might, but I'd rather not change it right now.